mava/systems/sable/sebulba/ff_sable.py

```diff
@@ -498,12 +465,11 @@ def learner_setup(
     opt_state = optim.init(params)
 
     # Pack apply and update functions.
-    apply_fns = (
-        partial(
-            sable_network.apply, method="get_actions"
-        ),  # Execution function required for the advantage calculation
-        partial(sable_network.apply),  # Training function
-    )
+    net_act_fn = partial(
+        sable_network.apply, method="get_actions"
+    )  # Required for the advantage calculation
```

Comment:
Sorry for the nit on my own change, I hate it when comments make things span multiple lines
```suggestion
    # Required for the advantage calculation
    net_act_fn = partial(sable_network.apply, method="get_actions")  
```